style/app.scss

```diff
@@ -687,131 +642,121 @@ a:active
-  position         : absolute;
-  padding-left     : 23px;
+  display          : flex;
+  flex-direction   : column;
+  z-index          : 1;
+  position         : inherit;
+  width            : 140px;
+  background-color : #ffffff;
+  border-radius    : 20px;
+  box-shadow       : 2px 2px 2px #e8e9eb;
+  margin-top       : 20px;
 }
 
-#sidebar-button
+.expanded
 {
-  cursor           : pointer;
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #5c497e;
-  position         : absolute;
-  border           : solid 1px #000000;
+  visibility : visible;
 }
 
-#sidebar-button:hover
+.expanded > .sidebar-dropdown
 {
-  background-color : #cac4d4;
+  height           : 350px;
+  padding          : 10px;
 }
 
-#sidebar-icon
+.expanded > .sidebar-button > #sidebar-icon
 {
-  width        : 30px;
-  height       : 50px;
-  padding-top  : 20px;
-  padding-left : 2px;
-  position     : absolute;
-  top          : 40%;
-  left         : 4px;
+  -webkit-transform : rotate(180deg);
+  -moz-transform    : rotate(180deg);
+  -ms-transform     : rotate(180deg);
+  -o-transform      : rotate(180deg);
+  transform         : rotate(180deg);
 }
 
-#focuses-nav, #graph-nav
+div#sidebar.collapsed
 {
-  cursor : pointer;
+  height: 40px;
 }
 
-#sidebar-nav
+.collapsed > .sidebar-dropdown
 {
-  width            : 100%;
-  font-size        : 13px;
-  background-color : #cac4d4;
-  border           : solid 1px #dedede;
-  box-shadow       : 0 2px 2px -1px rgba(0, 0, 0, 0.055);
-  display          : block;
-  overflow         : hidden;
+  display: none;
+  visibility: hidden;
 }
 
-#sidebar-nav ul
+.sidebar-button
 {
-  width  : 100%;
-  margin : 0 0 0 0;
+  cursor           : pointer;
+  border-radius    : 10px;
+  width            : 30px;
+  height           : 15px;
+  background-color : #d9d9d9;
+  margin-top       : -5px;
+  margin-left      : 60px;
+  text-align       : center;
 }
 
-#sidebar-nav ul li
+.selected-courses
 {
-  list-style-type    : none;
-  display            : inline-block;
-  width              : 48%;
-  -webkit-transition : all 0.2s;
-  -moz-transition    : all 0.2s;
-  -ms-transition     : all 0.2s;
-  -o-transition      : all 0.2s;
-  transition         : all 0.2s;
+  margin-top       : 0px;
+  margin-bottom    : 10px;
+  font-size        : 12px;
+  color            : #a47cb8;
 }
 
-#sidebar-nav ul li:hover
+.courses
 {
-  background-color : #5C497E !important;
+  overflow         : auto;
+  max-height       : 275px;
+  max-width        : 120px;
+  display          : inline-block;
 }
 
-#sidebar-nav ul li:hover a
+.course-selection
 {
-  color : white !important;
+  font-size  : 12px;
 }
 
-#sidebar-nav ul li .nav-section
+.sidebar-button:hover
 {
-  color           : #000000;
-  display         : inline-block;
-  line-height     : 30px;
-  text-align      : center;
-  width           : 95%;
-  text-decoration : none;
+  background-color: #cac4d4;
```

Comment:
This colour is too similar to the button text. Make the button text darker and this lighter.

Reply:
I'm assuming that this is actually in reference to the reset selections button, and not the sidebar button? However this could be me misunderstanding things again.

Reply:
Yes it was in reference to the reset selections button, but overall I wanted a greater contrast in both buttons, and you made good changes for both, so no need to make further colour changes.